hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -177,6 +177,11 @@ The gateway address to use if the TFTP server is on a differenet subnet e.g. "19
 Default: ""  
 Version: pieeprom-2020-04-16.bin  
 
+#### MAC_ADDRESS
+Overrides the Ethernet MAC address with the given value. e.g. dc:a6:32:01:36:c2`   
```

Comment:
There's a stray backtick at the end of this line.

Reply:
done